android/app/src/main/java/org/fossasia/openevent/adapters/FeedAdapter.java

```diff
@@ -107,7 +107,10 @@ public void onBindViewHolder(RecyclerViewHolder holder, int position) {
         }
 
         if (!TextUtils.isEmpty(feedItem.getMessage())) {
-            holder.statusMsg.setText(feedItem.getMessage());
+            String statusMsg = feedItem.getMessage();
+            if (feedItem.getLink() != null)
+                statusMsg = statusMsg.replace(feedItem.getLink(), "");
```

Comment:
Can you please add a comment regarding the reason for replacing the string for future developers? :wink: